apps/gallery/utils.py

```diff
@@ -0,0 +1,33 @@
+from django.conf import settings
+from django.urls import reverse
+from django.utils.encoding import force_bytes
+from django.utils.http import urlsafe_base64_encode
+
+from gallery.enums import PrivateFileModuleType
+from deep.permissions import ProjectPermissions as PP
+from lead.models import Lead
+
+
+def get_private_file_url(private_file_module_type: PrivateFileModuleType, entry):
```

Comment:
This is a generic function, we shouldn't pass entry here

```suggestion
def get_private_file_url(private_file_module_type: PrivateFileModuleType, id: int, filename: str):
```